fastly/kinesis.go

```diff
@@ -0,0 +1,242 @@
+package fastly
+
+import (
+	"fmt"
+	"net/url"
+	"sort"
+	"time"
+)
+
+// Kinesis represents a Kinesis response from the Fastly API.
+type Kinesis struct {
+	ServiceID      string `mapstructure:"service_id"`
+	ServiceVersion int    `mapstructure:"version"`
+
+	Name              string     `mapstructure:"name"`
+	StreamName        string     `mapstructure:"topic"`
+	Region            string     `mapstructure:"region"`
+	AccessKey         string     `mapstructure:"access_key"`
+	SecretKey         string     `mapstructure:"secret_key"`
+	Format            string     `mapstructure:"format"`
+	FormatVersion     uint       `mapstructure:"format_version"`
+	ResponseCondition string     `mapstructure:"response_condition"`
```

Comment:
I can't find `response_condition` defined in the API documentation?  https://developer.fastly.com/reference/api/logging/kinesis/

Is this new/awaiting to be added to the developer api reference page?

Reply:
I think this may be an accidental omission in the API. Every other logging endpoint has this field so let me check on this.

Reply:
Just in case you miss it further down, I did wonder if that was the case but I also noticed that your recorded fixture file also doesn't include it 🤔

Reply:
I have left this in since it supposed to be there and its benign I think to leave it in since Kinesis is only in LA right now. If you would rather remove it now and add it back in later I can do that too. Just let me know if that's your preference.